pwndbg/heap/__init__.py

```diff
@@ -17,7 +17,7 @@
     "global_max_fast", "0", "&global_max_fast for heuristics", "heap"
 )
 
-symbol_list = pwndbg.config.get_params("heap")
+symbol_list = [local for local in locals().values() if isinstance(local, pwndbg.config.Parameter)]
```

Comment:
Is this just all above or is there more to it? Can we just list it manually? Doing it with `locals()` can be problematic with further code changes

Reply:
> Is this just all above or is there more to it?

Yes, we only need above config for now.

> Can we just list it manually?

Yes. (I did this because we might need to modify this list every time we want to introduce a new symbol to let the user set it, so I want to avoid this, but seems this `locals()` solution is not good haha.)

> Doing it with `locals()` can be problematic with further code changes.

Ok, I'll just list it manually.